src/test/java/domain/PurchasedLottosTest.java

```diff
@@ -0,0 +1,55 @@
+package domain;
+
+import domain.strategy.CustomPurchaseStrategy;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+@DisplayName("상금과 수익률을 옳바르게 계산하는지 테스트한다.")
+public class PurchasedLottosTest {
+
+    PurchasedLotto lottos;
+    WinningNumber winningLotto;
+    int inputMoney;
+
+    @BeforeEach
+    void init() {
+        inputMoney = 1400;
+        lottos = new PurchasedLotto(inputMoney);
+        lottos.purchase(new CustomPurchaseStrategy(Arrays.asList(new Integer[]{1, 2, 3, 43, 44, 45})));
```

Comment:
```suggestion
        lottos.purchase(new CustomPurchaseStrategy(Arrays.asList(1, 2, 3, 43, 44, 45)));
```
`Arrays.asList`를 쓰셨던 만큼 위와 같이 쓰실수도 있어요~ Java 9 부터는 `List.of`도 사용할 수 있어서 참고해주시면 될거 같아요. (`List.of` 사용을 더 권장합니다)

참고: https://stackoverflow.com/questions/46579074/what-is-the-difference-between-list-of-and-arrays-aslist